src/components/PageTemplate/Navbar.jsx

```diff
@@ -14,79 +14,95 @@ import {
-                            <HamburgerIcon w={6} h={6}/>
-                        </MenuButton>
-                        <MenuList bg={bgColor}>
-                            <Link to="/"><MenuItem>Home</MenuItem></Link>
-                            <Link to="/watchlist"><MenuItem>Watchlist</MenuItem></Link>
-                            <Link to="/news"><MenuItem>News</MenuItem></Link>
-                            <Link to="/about"><MenuItem>About</MenuItem></Link>
-                        </MenuList>
-                    </Menu>
-                </Flex>
+const HamburgerMenu = ({bgColor}) => (
+    // visible when screen width narrow
+    <Flex display={['flex', 'flex', 'none', 'none']}>
+        <Menu>
+            <MenuButton as={Button} bg="transparent" px={PADDING} m={MARGIN}>
+                <HamburgerIcon w={ICON_SIZE} h={ICON_SIZE}/>
+            </MenuButton>
+            <MenuList bg={bgColor}>
+                <Link to="/"><MenuItem>Home</MenuItem></Link>
+                <Link to="/watchlist"><MenuItem>Watchlist</MenuItem></Link>
+                <Link to="/news"><MenuItem>News</MenuItem></Link>
+                <Link to="/about"><MenuItem>About</MenuItem></Link>
+            </MenuList>
+        </Menu>
+    </Flex>
+)
 
-                {/*TODO InvestoBull Logo*/}
-                <Link to="/"><Button fontSize={24} fontWeight={200} m={MARGIN}>InvestoBull</Button></Link>
+const Links = () => (
+    // visible when screen width wide
+    <Flex display={['none', 'none', 'flex', 'flex']}>
+        <Link to="/"><Button bg="transparent" m={MARGIN}>Home</Button></Link>
+        <Link to="/watchlist"><Button bg="transparent" m={MARGIN}>Watchlist</Button></Link>
+        <Link to="/news"><Button bg="transparent" m={MARGIN}>News</Button></Link>
+        <Link to="/about"><Button bg="transparent" m={MARGIN}>About</Button></Link>
+    </Flex>
+)
 
-                {/*Button Links, visible when screen width wide*/}
-                <Flex display={['none', 'none', 'flex', 'flex']}>
-                    <Link to="/"><Button bg="transparent" m={MARGIN}>Home</Button></Link>
-                    <Link to="/watchlist"><Button bg="transparent" m={MARGIN}>Watchlist</Button></Link>
-                    <Link to="/news"><Button bg="transparent" m={MARGIN}>News</Button></Link>
-                    <Link to="/about"><Button bg="transparent" m={MARGIN}>About</Button></Link>
-                </Flex>
+const ThemeSwitch = ({tcm}) => (
+    <Flex display={['none', 'flex', 'flex', 'flex']} p={PADDING} m={MARGIN}><Switch onChange={tcm}/></Flex>
+)
 
-                <Flex flex="1" align="center" justify="flex-end">
-                    {/*Light/Dark Mode Toggle*/}
-                    <Flex display={['none', 'flex', 'flex', 'flex']} p={1} m={MARGIN}>
-                        <Switch onChange={toggleColorMode} />
-                    </Flex>
+const Notifications = ({bgColor}) => (
```

Comment:
are you able to implement a dummy example of what the notifications would look like if there were unread notifications?
namely:
1. the red badge showing # of unread notifications
2. the un-clicked notifications in the modal having a light blue dot beside them
3. making the clicked notifications appear slightly lighter than the un-clicked ones

Reply:
@paulsgh this menu is merely a placeholder, I thought the notifications component would be a separate PR, but yea I can make something